OmniAX/Sources/Core/AX.swift

```diff
@@ -203,7 +203,7 @@ public final class AX: NSObject {
 
     /// Post an accessibility notification, focusing on/announcing the included focus item
     public static func post(notification: UIAccessibility.Notification, focus: Any?) {
-        guard !notification.isVoiceOverSpecific || voiceOverEnabled else {
+        guard !notification.isVoiceOverSpecific else {
```

Comment:
Let's fully remove this guard statement (might be able to remove the definition of `isVoiceOverSpecific` as well if becomes unused.